examples/hardhat-truffe-v5/package.json

```diff
@@ -14,7 +14,7 @@
     "@nomiclabs/hardhat-etherscan": "^2.1.1",
     "@nomiclabs/hardhat-truffle5": "^2.0.0",
     "@nomiclabs/hardhat-web3": "^2.0.0",
-    "@typechain/hardhat": "workspace:^7.0.0",
+    "@typechain/hardhat": "workspace:^8.0.0",
```

Comment:
why change this?

Reply:
My bad, this is a leftover of when I was trying things locally with verdaccio. I'll undo it.

Reply:
@fvictorio note i fixed this and all other issues in: https://github.com/dethcrypto/TypeChain/pull/845